app/controllers/api/v1/users_controller.rb

```diff
@@ -1,61 +1,15 @@
 class Api::V1::UsersController < Api::V1::BaseController
-  # before_action :admin_access_required, only: [:index]
+  before_action :ensure_admin_user, only: [ :index ]
 
   def index
-    if current_user.admin?
-      users = []
-      User.all.order(:id).each{|u| users << sanitize_user_attributes(u)}
-
-      render json: {users: users}
-    else
-      render json: {errors: "access denied"}, status: 403
-    end
+      users = User.all.order(:id)
+      render json: users, serializer: UserSerializer
```

Comment:
I'm going to sync up with the front end people to make sure we're all on the same page about the format of responses and making them all consistent. i have a feeling right now that everything is inconsistent but working, so we need to coordinate a time to make everything consistent on both ends.